src/libpgexporter/prometheus.c

```diff
@@ -1927,7 +1927,8 @@ histogram_information(struct prometheus* prom, int client_fd)
    int ret;
    int n_bounds = 0;
    int n_buckets = 0;
-   int histogram_start_idx;
+   int histogram_idx;
+   char* names[MAX_ARR_LENGTH];
```

Comment:
We don't need `names` anymore

Reply:
I now use `names` in `get_column_by_name` only.